lib/modules/betteReddit.js

```diff
@@ -200,6 +205,9 @@ addModule('betteReddit', function(module, moduleID) {
 				default:
 					break;
 			}
+			if (module.options.truncateLongLinks.value) {
+				RESUtils.addCSS('overflow: hidden;text-overflow: ellipsis;white-space: nowrap;');
```

Comment:
No, moving your changes into betteReddit.scss and wrapping them in `.res-betteReddit-truncateLongLinks { ... }`